wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationMessageSectionControllerTests.swift

```diff
@@ -86,14 +86,14 @@ final class ConversationMessageSectionControllerTests: XCTestCase {
         let section = ConversationMessageSectionController(
             message: MockMessage(),
             context: context,
-            userSession: userSession
+            userSession: userSession,
+            useInvertedIndices: true
         )
-        section.cellDescriptions.removeAll()
-        section.useInvertedIndices = true
```

Comment:
`useInvertedIndices ` was never changed at runtime, in fact if it was, the layout would not update correctly.
That's why I made the property private and added the argument for `init`.